app/controllers/profiles_controller.rb

```diff
@@ -0,0 +1,7 @@
+class ProfilesController < ApplicationController
```

Comment:
Should this be singular? http://guides.rubyonrails.org/routing.html#singular-resources seems like somebody would only have a single profile and an `#index` method doesn't make a lot of sense.